managers.py

```diff
@@ -0,0 +1,40 @@
+import sqlite3
+from models import Actor
+
+class ActorManager:
+    def __init__(self):
+        self._connection = sqlite3.connect("cinema.sqlite")
+        self.table_name = "actors"
+
+
+    def all(self):
+        cursor = self._connection.execute(
+            "SELECT * FROM actors"
+        )
+        rows = cursor.fetchall()
+        actors = [Actor(*row) for row in rows]
+        return actors
+
+
+    def create(self, first_name: str, last_name: str):
+        self._connection.execute(
+            f"INSERT INTO {self.table_name} (first_name, last_name) VALUES (?, ?)",
+            (first_name, last_name)
+        )
+        self._connection.commit()
+
+
+    def update(self, id_to_update: int, new_first_name: str, new_last_name: str):
+        self._connection.execute(
+            f"UPDATE {self.table_name} SET first_name = ?, last_name = ? WHERE id = ?",
+            (new_first_name, new_last_name, id_to_update)

```

Comment:
It's better to follow SQL style guidelines even while writing raw SQL in Python. Therefore I'd suggest to start FROM clause from new line.
Fix it everywhere : )